repository-data/webfiles/src/main/resources/site/freemarker/intranet/task.ftl

```diff
@@ -68,18 +72,26 @@
     </#list>
 </#if>
 
-<article class="article article--intranet-task">
-
+<article>
     <#assign metaData = [] />
     <#if document.topics?has_content>
         <#assign metaData += [{"key":"Topics", "value": document.topics?join(", "), "uipath": "taxonomy", "schemaOrgTag": "", "type": ""}] />
     </#if>
-    <@documentHeader document 'intranet-task' "" "" "" "" false metaData></@documentHeader>
+    <#assign heroOptions = getHeroOptions(document) />
+
+    <#assign digiblocks = ["tr","bl"] />
```

Comment:
Because this page has a columned layout (sticky nav & content) we would want to use a left-aligned hero with digiblocks on the right only. The left-aligned hero is the default layout so you should be able to remove these extra options and use the default options from `getHeroOptions(document)`.

Reply:
done

Reply:
I put it back, as the ticket says the digiblocks are required both sides

Reply:
Changed again